src/org/labkey/test/components/experiment/LineageGraph.java

```diff
@@ -0,0 +1,105 @@
+package org.labkey.test.components.experiment;
+
+import org.labkey.test.Locator;
+import org.labkey.test.components.Component;
+import org.labkey.test.components.WebDriverComponent;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class LineageGraph extends WebDriverComponent<LineageGraph.ElementCache>
+{
+    final WebElement _el;
+    final WebDriver _driver;
+
+    public LineageGraph(WebElement element, WebDriver driver)
```

Comment:
Make constructor protected since only the `Finder` knows how to find it.
```suggestion
    protected LineageGraph(WebElement element, WebDriver driver)
```